godot-ffi/src/lib.rs

```diff
@@ -77,39 +76,278 @@ pub use toolbox::*;
 // ----------------------------------------------------------------------------------------------------------------------------------------------
 // API to access Godot via FFI
 
+mod binding {
+    // Ensure both crates are checked regardless of cfg, for the sake of development convenience.
+    #[cfg_attr(not(feature = "experimental-threads"), allow(dead_code))]
+    mod multi_threaded;
+    #[cfg_attr(feature = "experimental-threads", allow(dead_code))]
+    mod single_threaded;
```

Comment:
Not a huge deal, but the problem is that this allocates additional `static`s even in the unused configuration. 

Or do you know if those are optimized out?

Reply:
They should be optimized out if they're not accessed (which they aren't), i can try to check if that is the case though.

Reply:
From what i can tell, it seems like there are no references at all to the module that is not active in the assembly, barring debug info if that is enabled. (which it isn't for release builds by default). So yes it seems that they are actually optimized out.